tests/python_package_test/test_engine.py

```diff
@@ -1072,6 +1073,64 @@ def test_cvbooster():
     assert ret < 0.15
 
 
+def test_cvbooster_save_load(tmp_path):
```

Comment:
Can we also check that `best_iteration` attribute was saved and loaded correctly?

Reply:
@StrikerRUS 
I’ve added the test in 26365003532865bdf751684f7beb06b4074279e2.